src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/ViolationTicketCount.java

```diff
@@ -38,83 +39,89 @@ public class ViolationTicketCount extends Auditable<String> {
 	@Schema(description = "ID", accessMode = Schema.AccessMode.READ_ONLY)
 	@Id
 	@GeneratedValue
-    private Long violationTicketCountId;
+	private Long violationTicketCountId;
 
 	/**
 	 * The count number.
 	 */
 	@Column
-	@Min(1) @Max(3)
+	@Min(1)
+	@Max(3)
 	private int countNo;
 
 	/**
 	 * The description of the offence.
 	 */
 	@Column(length = 4000)
 	@Schema(nullable = true)
-    private String description;
+	private String description;
 
 	/**
 	 * The act or regulation code the violation occurred against. For example, MVA, WLA, TCR, etc
 	 */
 	@Column(length = 5)
 	@Schema(nullable = true)
-    private String actOrRegulationNameCode;
+	private String actOrRegulationNameCode;
 
 	/**
 	 * The count is flagged as an offence to an act. Cannot be true, if is_regulation is true.
 	 */
 	@Column
 	@Schema(nullable = true)
 	@Enumerated(EnumType.STRING)
-    private YesNo isAct;
+	private YesNo isAct;
 
 	/**
 	 * The count is flagged as an offence to a regulation. Cannot be true, if is_act is true.
 	 */
 	@Column
 	@Schema(nullable = true)
 	@Enumerated(EnumType.STRING)
-    private YesNo isRegulation;
+	private YesNo isRegulation;
 
 	/**
 	 * The section part of the full section. For example, "127"
 	 */
-	@Column(length = 15)
-	@Schema(nullable = true, accessMode = Schema.AccessMode.READ_ONLY)
-    private String section;
+	@Size(max = 10)
+	@Column(length = 10)
+	@Schema(nullable = true, maxLength = 10, accessMode = Schema.AccessMode.READ_ONLY)
```

Comment:
I'm not sure why we have defined these section, subsection etc. fields as READ_ONLY for swagger open api spec in the first place but I don't see any reason for them to be at the moment since they should be a part of PUT request at least.

Reply:
I think because this data is lookup data. All these fields make up a lookup reference for the statute code in the JUSTIN_STATUTES table. By rights we shouldn't even have these columns, only the statute code.  It's like finding a code table record by the code description instead of by the code ID.
But I think we are now not using the statute code and only have the description data, so yeah, we could probably drop the read_only attribute now.